pommerman/graphics.py

```diff
@@ -0,0 +1,327 @@
+    reraise(prefix="Error occured while running `from pyglet.gl import *`",suffix="HINT: make sure you have OpenGL install. On Ubuntu, you can run 'apt-get install python-opengl'. If you're running on a server, you may need a virtual frame buffer; something like this should work: 'xvfb-run -s \"-screen 0 1400x900x24\" python <your_script.py>'")
+
+
+IMAGES_DICT = {
+    0: { 'id': 0, 'file_name': 'Passage.png', 'name': 'Passage', 'image': None},
+    1: { 'id': 1, 'file_name': 'Rigid.png', 'name': 'Rigid', 'image': None},
+    2: { 'id': 2, 'file_name': 'Wood.png', 'name': 'Wood', 'image': None},
+    3: { 'id': 3, 'file_name': 'Bomb.png', 'name': 'Bomb', 'image': None},
+    4: { 'id': 4, 'file_name': 'Flames.png', 'name': 'Flames', 'image': None},
+    5: { 'id': 5, 'file_name': 'Fog.png', 'name': 'Fog', 'image': None},
+    6: { 'id': 6, 'file_name': 'ExtraBomb.png', 'name': 'ExtraBomb', 'image': None},
+    7: { 'id': 7, 'file_name': 'IncrRange.png', 'name': 'IncrRange', 'image': None},
+    8: { 'id': 8, 'file_name': 'Kick.png', 'name': 'Kick', 'image': None},
+    9: { 'id': 9, 'file_name': 'Skull.png', 'name': 'Skull', 'image': None},
+    10: { 'id': 10, 'file_name': 'AgentDummy.png', 'name': 'AgentDummy', 'image': None},
+    11: { 'id': 11, 'file_name': 'Agent0.png', 'name': 'Agent0', 'image': None},
+    12: { 'id': 12, 'file_name': 'Agent1.png', 'name': 'Agent1', 'image': None},
+    13: { 'id': 13, 'file_name': 'Agent2.png', 'name': 'Agent2', 'image': None},
+    14: { 'id': 14, 'file_name': 'Agent3.png', 'name': 'Agent3', 'image': None},
+    15: { 'id': 15, 'file_name': 'AgentDummy-No-Background.png', 'name': 'AgentDummy-No-Background', 'image': None},
+    16: { 'id': 16, 'file_name': 'Agent0-No-Background.png', 'name': 'Agent0-No-Background', 'image': None},
+    17: { 'id': 17, 'file_name': 'Agent1-No-Background.png', 'name': 'Agent1-No-Background', 'image': None},
+    18: { 'id': 18, 'file_name': 'Agent2-No-Background.png', 'name': 'Agent2-No-Background', 'image': None},
+    19: { 'id': 19, 'file_name': 'Agent3-No-Background.png', 'name': 'Agent3-No-Background', 'image': None},
+    20: { 'id': 20, 'file_name': 'X-No-Background.png', 'name': 'X-No-Background', 'image': None}
+}
+
+FONTS_FILE_NAMES = [
+    'Cousine-Regular.ttf'
+]
+
+BORDER_SIZE = 20
+MARGIN_SIZE = 10
+TILE_SIZE = 50
+RESOURCE_DIR = 'resources/'
+BACKGROUND_COLOR = (41, 39, 51, 255)
+TILE_COLOR = (248, 221, 82, 255)
+TEXT_COLOR = (170, 170, 170, 255)
+
+HUMAN_FACTOR = 32
+AGENT_COLORS = [[231,76,60], [46,139,87], [65,105,225], [238,130,238]] # color for each of the 4 agents
+ITEM_COLORS = [[240,248,255], [128,128,128], [210,180,140], [255, 153, 51], [241, 196, 15], [141, 137, 124]]
+ITEM_COLORS += [(153, 153, 255), (153, 204, 204), (97, 169, 169), (48, 117, 117)] # ExtraBomb, IncrRange, etc. 
+
+__location__ = os.path.dirname(os.path.realpath(__file__))
+_resource_path = os.path.join(__location__, RESOURCE_DIR)
+
+class PixelViewer(object):
```

Comment:
It's my fault for not specifying this somewhere: within a class use one line between functions (; outside of a class, use two).